docs/releases/1.10.txt

```diff
@@ -732,6 +732,17 @@ use the default_related_name ``bars``::
 
     >>> Foo.object.get(bars=bar)
 
+Custom related manager must define a ``_apply_rel_filters()`` method
+--------------------------------------------------------------------
+
+If you defined a custom manager class available through
+:meth:`~django.db.models.query.QuerySet.prefetch_related` you must make sure
```

Comment:
Custom manager classes available through prefetch_related must define a ...
(not sure if an example is needed or if this is advanced enough that anyone doing this will know what it means?)